node/src/app.ts

```diff
@@ -26,8 +27,15 @@ export async function createApp(): Promise<Express> {
   if (process.env['NODE_ENV'] !== 'test') {
     app.use(morgan('dev'));
   }
-  app.use(cors());
+  //app.use(cors());
+  const corsOptions = {
```

Comment:
This seems like something we don't want in our production code. Why are we doing this?

If we did keep it, should be inlined into `app.use(cors({...})` below

Reply:
We would need to set the cors option if we want Axios to send cookie (refresh token) back to the backend.

1.to do that Axios says we need to use withCredentials: true in frontend. If we do that and do not change cors then we get an error 'The Access-Control-Allow-Origin cannot be a wildcard if credentials are being used'. Access-Control-Allow-Origin=* when we do the only app.use(cors())